tests/translator/input/api_with_resource_policy_global.yaml

```diff
@@ -0,0 +1,36 @@
+Globals:
+  Api:
+    Auth:
+      ResourcePolicy:
+        CustomStatements: [{
+            Action: 'execute-api:Invoke', 
+            Resource: ['execute-api:/*/*/*']
+          }]
+
+Resources:
+ ExplicitApi:
```

Comment:
So custom statements will override an explicitly defined OpenApi doc? It's not strictly limited to implicit APIs? I guess we're doing that elsewhere with auth specifically, but just wanted to confirm. Also, should we have an example with implicit API only (lambda function implies an API, globals adds custom statements to resource policy)?

Reply:
Yes, I think this applies to all the auth properties in general. 

Added implicit-only example.